public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -412,6 +412,9 @@ code-example(format='').
   making private variables for both:
 +makeExample('toh-5/ts/app/hero-detail.component.ts', 'ctor', 'app/hero-detail.component.ts (constructor)')(format=".")
 :marked
+  We implement the `OnInit` interface.
++makeExample('toh-5/ts/app/hero-detail.component.ts', 'implement', 'app/hero-detail.component.ts (class)')(format=".")
```

Comment:
If we keep this, please leave out the blue title bar that reads `app/hero-detail.component.ts (class)`.

We're not showing the class, just this one line of the class. I don't think we need the title bar at all.